benchto-driver/src/main/java/io/trino/benchto/driver/execution/QueryExecutionDriver.java

```diff
@@ -104,11 +101,22 @@ private QueryExecutionResult executeSelectQuery(
         }
     }
 
-    private QueryExecutionResult executeUpdateQuery(Connection connection, QueryExecutionResultBuilder queryExecutionResultBuilder, String sqlStatement)
+    private QueryExecutionResult executeUpdateQuery(
+            Connection connection,
+            QueryExecutionResultBuilder queryExecutionResultBuilder,
+            String sqlStatement,
+            Optional<Path> resultFile)
```

Comment:
Its optional because `query-results` config is also optional, so by default no validation is performed.
Only when user sets up this folder with validation files and point to this folder using this config validation will be performed.